.travis.yml

```diff
@@ -16,6 +16,7 @@ script:
   - npm run eslint
   - npm run stylelint
   - npm test
+  - npm run test:a11y
```

Comment:
- npm run test:a11y
- npm test

they should in this order, otherwise the report coverage is very low

Reply:
@youluna fixed